### PR TITLE
Extracting embeddings from graph and saving them

### PR DESCRIPTION
**Pull request by evsmithx**
Created a script to load embeddings and vocab from files so that the embeddings can be used elsewhere.

### CODE REVIEW

File: libs/ml/examples/word2vec_embeddings_extraction/main.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+#include "ml/dataloaders/word2vec_loaders/sgns_w2v_dataloader.hpp"
+#include "ml/exceptions/exceptions.hpp"
+#include "ml/utilities/word2vec_utilities.hpp"
+
+#include <string>
+
+using namespace fetch::ml;
+using namespace fetch::ml::dataloaders;
+
+using DataType   = float;
```
Comment (baykaner) [inline libs/ml/examples/word2vec_embeddings_extraction/main.cpp]: fixed point!

File: libs/ml/examples/word2vec_embeddings_extraction/main.cpp [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+using TensorType = fetch::math::Tensor<DataType>;
+using SizeType   = fetch::math::SizeType;
+
```
Comment (baykaner) [inline libs/ml/examples/word2vec_embeddings_extraction/main.cpp:33]: add helper function that takes in word as string and outputs vector from embedding matrix

File: libs/ml/examples/word2vec_graph_evaluation/main.cpp [hunk 3]
```diff
@@ -16,70 +16,120 @@
...
-#include <stdexcept>
 #include <string>
 
 using namespace fetch::ml;
 using namespace fetch::ml::dataloaders;
 
-using DataType   = float;
+// Note: DataType needs to be the same as that used for Graph if -graph option is specified
+using DataType = float;
+// using DataType   = fetch::fixed_point::FixedPoint<32, 32>;
```
Comment (baykaner) [inline libs/ml/examples/word2vec_graph_evaluation/main.cpp]: lets default to fixed point everywhere

**Review by baykaner — APPROVED** [review]

**Review by lrahmani — APPROVED** [review]
LGTM

**Pull request merged**